app/models/person_escort_record.rb

```diff
@@ -74,6 +75,20 @@ def build_responses!
     self
   end
 
+  def import_nomis_mappings!
```

Comment:
If this does not perform very well, we can send to a sidekiq job, but this needs to be coordinated with the FE to show the correct messaging and polling